bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -42,3 +43,16 @@ data:
     hybrid-overlay-vxlan-port="{{.OVNHybridOverlayVXLANPort}}"
     {{- end }}
 {{- end  }}
+{{- if .IsSNO }}
+
+    [masterha]
+    {{- /* 
+    Even in case of SNO there will be only one ovn-master, we set dedicated values for leader election 
```

Comment:
According the [slack conversation](https://coreos.slack.com/archives/CDCP2LA9L/p1627477771126100?thread_ts=1627402405.090600&cid=CDCP2LA9L) there are concerns around increasing the leader election times as well, but going ahead with this option as the current way forward.